proto/composerapis/astria/auctioneer/v1alpha1/bundle.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package astria.auctioneer.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+import "astria/execution/v1alpha2/execution.proto";
+import "astria/auctioneer/v1alpha1/optimistic_block.proto";
+
+// Information for the bundle submitter to know how to submit the bundle.
+// The fee and base_sequencer_block_hash are not necessarily strictly necessary
+// it allows for the case where the server doesn't always send the highest fee
+// bundles after the previous but could just stream any confirmed bundles.
+message Bundle {
+    // The fee that can be expected to be received for submitting this bundle.
+    // This allows the bundle producer to stream any confirmed bundles they would be ok
+    // with submitting. Used to avoid race conditions in received bundle packets. Could
+    // also be used by a bundle submitter to allow multiple entities to submit bundles.
+    uint64 fee = 1;
+    // The byte list of transactions to be included.
+    repeated bytes transactions = 2;
+    // The base_sequencer_block_hash is the hash from the base block this bundle
+    // is based on. This is used to verify that the bundle is based on the correct
+    // Sequencer block.
+    bytes base_sequencer_block_hash = 3;
```

Comment:
who's verifying it?

Reply:
@itamarreif i believe here, auctioneer could use the rollup block hash instead of sequencer block hash? cause if this is the case, rollup would have to maintain the sequencer_block_hash?
I believe auctioneer would anyways receive via the `ExecuteOptimisticBlock`

Reply:
I'm unsure this is needed here. the bundles will be mapped to an auction based on the rollup parent hash. the rollup ends up reading these bundle objects in the read path, and it currently is not aware of any sequencer-level hashes.

It might make sense to return the sequencer block hash alongside the optimistic execution result tho. i.e.
```protobuf
message OptimisticExecutionResult {
    bytes sequencer_block_hash = 1;
    astria.execution.v1alpha2.Block execution_result = 2;
}

service BundleService {
    rpc ExecuteOptimisticBlocks(stream BaseBlock) returns (stream OptimisticExecutionResult);
}
```

Reply:
we can make geth send this, if this helps with auctioneer